src/components/AccountDrawer/MiniPortfolio/Activity/ActivityRow.tsx

```diff
@@ -52,7 +53,7 @@ export function ActivityRow({
         descriptor={
           <ActivityRowDescriptor color="textSecondary">
             {descriptor}
-            {ENSName ?? otherAccount}
+            {ENSName ?? shortenAddress(otherAccount ?? '')}
```

Comment:
nit: instead of passing in an empty string if otherAcount is undefined, you can check beforehand ie:
`{otherAccount && (ENSName ?? shortenAddress(otherAccount))}`

(you'll know ENSName is undefined if otherAccount is also undefined)

Reply:
this one less sure of, reads nicer i think as is and shouldn't cost any performance really